open_academy/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+{
+    'name': "Open Academy",
+
```

Comment:
Please remove all blank lines between keys.

```suggestion
'name': "Open Academy",
'summary': """
    Module description""",
```